src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -30,9 +30,9 @@ export interface DurationSpec {
 
 export interface ContainerSpec {
   description: string | null;
-  capacity: QuantitySpec | null;
-  minimum_volume: MinimumVolumeSpec | null;
-  cap: Code | null;
+  capacity?: QuantitySpec | null;
+  minimum_volume?: MinimumVolumeSpec | null;
+  cap?: Code | null;
   preparation: string | null;
```

Comment:
```suggestion
  preparation?: string | null;
```

<img width="670" alt="image" src="https://github.com/user-attachments/assets/60645d37-964f-4be1-a1db-05fafee2f44c" />